### PR TITLE
fix(server_status): format it properly

### PR DESCRIPTION
**Pull request by diamante0018**
bot0 required for consistency between our clients and plutonium as the bot's guid is usually bot%d (%d replaced by qport)
This is not possible currently so just like I've done on plutonium t5 it's just bot0
Then i fixed the status header

### CODE REVIEW

File: src/client/component/status.cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	struct component final : generic_component
+	{
+		void post_unpack() override
+		{
+			// Patch the status command for test clients
+			utils::hook::call(game::select(0x142246E37, 0x14052C527), print_client_num);
+			utils::hook::call(game::select(0x142246EDE, 0x14052C5CE), print_client_xuid);
+
+			utils::hook::copy_string(game::select(0x143050480, 0x140E85A20),
+			                         "num score ping guid             name             address                  qport  \n");
```
Comment (momo5502) [inline src/client/component/status.cpp:40]: original status says xuid, doesn't it?

Comment (diamante0018) [reply src/client/component/status.cpp:40]: our clients have guid. storm asked me to rename but you would be correct. the original status says xuid

Comment (momo5502) [reply src/client/component/status.cpp:40]: note there is two status handlers in the client, one that says guid and the one that says xuid. i think the one with xuid is the one that's actively used. so make sure you base this on the proper on

Comment (diamante0018) [reply src/client/component/status.cpp:40]: I made sure this is the one that prints into the console when i type status command in my in-game console

Comment (diamante0018) [reply src/client/component/status.cpp]: i renamed it back to xuid so it stays consistent. but this should be the correct status handler between the two

Comment (momo5502) [reply src/client/component/status.cpp]: perfect, just wanted to make sure.

**Review by momo5502 — APPROVED** [review]

File: src/client/component/status.cpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+#include "loader/component_loader.hpp"
+#include <game/game.hpp>
+
+#include <utils/hook.hpp>
+
+namespace status
+{
+	namespace
+	{
+		int client_num_;
```
Comment (momo5502) [inline src/client/component/status.cpp:11]: i recommend this to be thread_local

**Review by momo5502 — CHANGES_REQUESTED** [review]

File: src/client/component/status.cpp (hunk 2 above)
Comment (momo5502) [reply src/client/component/status.cpp:11]: even though technically it's not really necessary,calling the status command from different threads concurrently could yield errors. we know this is not gonna happen, but using thread_local still seems to be a good practice in that case

Comment (diamante0018) [reply src/client/component/status.cpp]: done

**Review by momo5502 — APPROVED** [review]

**Pull request merged**